packages/preact-iso/package.json

```
@@ -12,6 +12,7 @@
     "./hydrate": "./hydrate.js"
   },
   "author": "The Preact Authors (https://preactjs.com)",
+  "repository": "preactjs/wmr",	
```

I just looked it up and apparently you can specify the directory too! (no shortform sadly)

```suggestion
  "repository": {
    "type" : "git",
    "url" : "https://github.com/preactjs/wmr",
    "directory": "packages/preact-iso"
  },
```